src/Clustering/Clustering.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2017 Google Inc.
```

Comment:
2019*

Reply:
Ok,  I will update and merge from master and copyright.